README.md

```diff
@@ -1,15 +1,47 @@
 # ansible-role-nginx
 
-A brief description of the role goes here.
+Manages `nginx`.
 
 # Requirements
 
 None
 
 # Role Variables
 
-| variable | description | default |
+| Variable | Description | Default |
 |----------|-------------|---------|
+| `nginx_user` | user of `nginx` | `{{ __nginx_user }}` |
+| `nginx_group` | group of `nginx` | `{{ __nginx_group }}` |
+| `nginx_log_dir` | path to log directory | `{{ __nginx_log_dir }}` |
+| `nginx_service` | service name of `nginx` | `nginx` |
+| `nginx_package` | package name of `nginx` | `{{ __nginx_package }}` |
+| `nginx_conf_dir` | path to configuration directory | `{{ __nginx_conf_dir }}` |
+| `nginx_conf_fragment_dir` | path to optional configuration fragment directory | `{{ nginx_conf_dir }}/conf.d` |
```

Comment:
i think `nginx_conf_fragments_dir` is better as it is for `nginx_config_fragments`.